File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocso/OCS2016GetLinkingDepartmentInfo.java
package nta.med.data.model.ihis.ocso;

public class OCS2016GetLinkingDepartmentInfo {

	private String departmentCode;
	private String departmentName;

	public OCS2016GetLinkingDepartmentInfo(String departmentCode, String departmentName) {
		super();
		this.departmentCode = departmentCode;
		this.departmentName = departmentName;
	}

	public String getDepartmentCode() {
		return departmentCode;
	}

	public void setDepartmentCode(String departmentCode) {
		this.departmentCode = departmentCode;
	}

	public String getDepartmentName() {
		return departmentName;
	}

	public void setDepartmentName(String departmentName) {
		this.departmentName = departmentName;
	}

	
	
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocso/OCS2016GetLinkingDepartmentInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocso
{
    [Serializable]
    public class OCS2016GetLinkingDepartmentInfo
    {
        private String _departmentCode;
        private String _departmentName;

        public String DepartmentCode
        {
            get { return this._departmentCode; }
            set { this._departmentCode = value; }
        }

        public String DepartmentName
        {
            get { return this._departmentName; }
            set { this._departmentName = value; }
        }

        public OCS2016GetLinkingDepartmentInfo() { }

        public OCS2016GetLinkingDepartmentInfo(String departmentCode, String departmentName)
        {
            this._departmentCode = departmentCode;
            this._departmentName = departmentName;
        }

    }
}